<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Multiplayer</class>
 <widget class="QWidget" name="Multiplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>160</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PLAYER 2: USERNAME</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>280</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PLAYER 2: PASSWORD</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="username">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>220</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>610</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/play.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>610</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/back.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>240</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>500</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QSplitter" name="levels">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>240</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="level1">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="level2">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="level3">
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QSplitter" name="characteres">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>240</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="character1">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="character2">
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="character3">
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
